src/main/resources/com/mathworks/ci/UseMatlabVersionBuildWrapper/help-matlabRootFolder.html

```diff
@@ -0,0 +1,8 @@
+<div>
+  Enter the full path to the MATLAB root folder, which is returned by the <i><b>matlabroot</b></i> function. The specified MATLAB will be added to PATH variable and will be used across this build.
+  If this field is empty then default matlabroot from PATH variable will be picked if added. <br><br><br>Example:<br><br><i><b><u>Windows:</u></b></i><br> C:\Program Files\MATLAB\R2019a <br>
```

Comment:
I would try to reword these sentences:

"The specified MATLAB will be added to PATH variable and will be used across this build.
If this field is empty then default matlabroot from PATH variable will be picked if added."

as

"The specified directory will be added to the front of the PATH environment variable, and the specified MATLAB version will be used to perform the build.
You must not leave this field blank if no MATLAB directory exists on the path. If you leave the field blank, then the system tries to use the MATLAB directory closest to the top of the system path."

Does it make sense?

Reply:
Yes its clear , however i have only one word suggestion let me know if we could use prefixed instead added to the front. which looks as below.

The specified directory will be prefixed to the  PATH environment variable, and the specified MATLAB version will be used to perform the build.
You must not leave this field blank if no MATLAB directory exists on the path. If you leave the field blank, then the system tries to use the MATLAB directory closest to the top of the system path.

I am making the changes as per your suggestion for review however if you agree on my suggestion i shall change it else will keep it as is and consider it approved.

Reply:
Hi Nikhil,

Thanks for your suggestion. I think we can use "prepended" instead of "prefixed". So the text would become:

The specified directory will be prepended to the PATH environment variable, and the specified MATLAB version will be used to perform the build.
You must not leave this field blank if no MATLAB directory exists on the path. If you leave the field blank, then the system tries to use the MATLAB directory closest to the top of the system path.

Thanks,
Houman

Reply:
Done.